Pcap++/header/PcapLiveDevice.h

```diff
@@ -211,6 +211,12 @@ namespace pcpp
 			*/
 			int snapshotLength;
 
+			/**
+			 * Set extra information. For some types of device, extra information might be required. Such a case is when opening a NFLOG 
+			 * device and setting a different NFLOG group. If no extra information is required, this configuration is ignored.
+			*/
+			unsigned int extra;
```

Comment:
Not important but if it is only related with NFLOG, the name of the parameter might include NFLOG. For example, like nflogFlags or nflogInfo etc

Reply:
The idea was that sometime in the future, some device might require some extra information as well. So to make it as generic as possible so any other device could use a piece of extra information if needed, I called it extra instead of NflogGroup.

Reply:
With this in mind, do you think I still should have the name as "nflogGroup"? Or should I keep it as "extra"?

Reply:
Both options are ok for me

Reply:
I vote for being more specific and calling this parameter `nflogGroup`. In the parameter doxy we can mention this is only relevant for nflog interfaces

Reply:
Alright. Done!